src/test/java/org/assertj/core/api/abstract_/AbstractAssert_satisfies_with_Consumers_Test.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2021 the original author or authors.
+ */
+package org.assertj.core.api.abstract_;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.api.BDDAssertions.thenIllegalArgumentException;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+
+import java.util.function.Consumer;
+
+import org.assertj.core.test.Jedi;
+import org.junit.jupiter.api.Test;
+
+class AbstractAssert_satisfies_with_Consumers_Test {
+  private final Jedi yoda = new Jedi("Yoda", "Green");
+
+  @Test
+  void should_pass_satisfying_single_requirement() {
+    // GIVEN
+    Consumer<Jedi> isNamedYoda = jedi -> assertThat(jedi.getName()).as("check yoda").isEqualTo("Yoda");
+    // WHEN-THEN
+    then(yoda).satisfies(isNamedYoda);
+  }
+
+  @Test
+  void should_pass_satisfying_multiple_requirements() {
+    // GIVEN
+    Consumer<Jedi> isNamedYoda = jedi -> assertThat(jedi.getName()).as("check yoda").isEqualTo("Yoda");
+    Consumer<Jedi> hasGreenLightSaber = jedi -> assertThat(jedi.lightSaberColor).as("check light saber").isEqualTo("Green");
+    // WHEN-THEN
+    then(yoda).satisfies(hasGreenLightSaber, isNamedYoda);
+  }
+
+  @Test
+  void should_fail_not_satisfying_single_requirement() {
+    // GIVEN
+    Consumer<Jedi> isNamedVader = jedi -> assertThat(jedi.getName()).as("check vader").isEqualTo("Darth Vader");
+    // WHEN
+    AssertionError assertionError = expectAssertionError(() -> assertThat(yoda).satisfies(isNamedVader));
+    // THEN
+    then(assertionError.getMessage()).contains("check vader");
```

Comment:
I noticed that even if there is only a single failing requirement, the error report contains message "Multiple failures...". Would it make sense to create a shortcut for the case when there is only one failure, so the error message would be more simple?